CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# dbt_xero_source v0.3.0
+
+## Features
+- Allow for multiple sources by unioning source tables across multiple Xero connectors.
+([#11](https://github.com/fivetran/dbt_xero_source/pull/11))
+  - Refer to the README for more details. (See ### Unioning Multiple Xero Connectors).
```

Comment:
Can you instead link to that section instead? 
```suggestion
  - Refer to the [README](https://github.com/fivetran/dbt_xero_source#unioning-multiple-xero-connectors) for more details.
```
FYI the link won't work until the PR is merged.